props/blaster.h

```diff
@@ -77,10 +83,53 @@ class Blaster : public PropBase {
       } else {
         hybrid_font.PlayCommon(&SFX_blast);
       }
+
+      shots_fired_++;
 #endif
     }
   }
 
+  virtual void Reload() {
+    shots_fired_ = 0;
+
+#ifdef ENABLE_AUDIO
+    if (SFX_reload.files_found()) {
+      hybrid_font.PlayCommon(&SFX_reload);
```

Comment:
Calling reload here is known as a "layer violation".
Basically it means that you're doing something in the wrong place.
It's supposed to be up to the hybrid font how to handle a reload event.
I'm ok with checking this in for now, because it's better than not handling reload at all, but the problem with this is that no events is sent to the blades, so it's not possible to add any effects to a reload events.